cpw/CPWBend.py

```diff
@@ -5,7 +5,14 @@
 import numpy as np
 
 class CPWBend(Component):
-    """A CPW bend"""
+    """
+    creates a CPW bend with pinw/gapw/radius
+            
+    turn_angle: turn_angle is in degrees, positive is CCW, negative is CW
+    polyarc: True/False, True draws CPWBend as a polyline, False as arcs and lines
```

Comment:
do we ever use the non polyline version?

Reply:
I specifically removed it this version, but kept it as a setting to "not mess with existing files" (I don't think its ever been used with the current MaskMaker). I can just take it out completely, but it seemed more correct to keep it compatible with designs from previous versions.

Reply:
next time